test/javascript/src/components/ChartCard/BarChartByTechnicalArea.test.js

```diff
@@ -15,6 +15,7 @@ const stateFromServerForInfluenza = fs.readFileSync(
 
 it("BarChartByTechnicalArea has the expected 2 divs", () => {
   const initialState = JSON.parse(stateFromServerForInfluenza)
+  initialState.ui = { selectedTechnicalAreaId: null }
```

Comment:
I think this is too much coupling, it should be stubbed out at this level and tested at the selectors level instead.